boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpunet.dts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf5340_cpunet_qkaa.dtsi>
+
+/ {
+	model = "Nordic NRF5340 DK NRF5340 Network";
+	compatible = "nordic,nrf5340-dk-nrf5340-cpunet";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram1;
+		zephyr,flash = &flash1;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 28 GPIO_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
+			label = "Green LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 30 GPIO_ACTIVE_LOW>;
+			label = "Green LED 2";
+		};
+		led3: led_3 {
+			gpios = <&gpio0 31 GPIO_ACTIVE_LOW>;
+			label = "Green LED 3";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 23 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 1";
+		};
+		button1: button_1 {
+			gpios = <&gpio0 24 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 2";
+		};
+		button2: button_2 {
+			gpios = <&gpio0 8 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 3";
+		};
+		button3: button_3 {
+			gpios = <&gpio0 9 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 4";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		led3 = &led3;
+		sw0 = &button0;
+		sw1 = &button1;
+		sw2 = &button2;
+		sw3 = &button3;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <25>;
+	rx-pin = <26>;
+	rts-pin = <10>;
+	cts-pin = <12>;
```

Comment:
The back of my 0.9.0 board has

    P0.19 RTS
    P0.20 TXD
    P0.21 CTS
    P0.22 RXD

    P0.11 RTS
    P1.01 TXD
    P0.10 CTS
    P1.00 RXD

Reply:
If so, new UART pin settings need to be assigned to the Network Core by the Application Core in the cpunet_reset.c module. Unfortunately, we don't have access to the cpunet DTS in the cpuapp build targets.

Reply:
Please use UART pins for the Network Core that were listed by @oyvindronningstad. They are linked with the first COM port. This way, we will have two COM ports active - one for Application Core logs and one for Network Core logs. See my test commit for reference:

https://github.com/kapi-no/fw-nrfconnect-zephyr/commit/a0e6320accea67900d6fb273b7041b4d51e1239d

Reply:
Note that I'm not sure what the pin info I listed means :P. I.e. which UARTs they refer to, and which core.

Reply:
Yes, but I think it is still sensible to leave the first set of UART pins for Application Core because pin configuration doesn't differ between the DK and PDK.

Reply:
>Please use UART pins for the Network Core that were listed by @oyvindronningstad. They are linked with the first COM port. This way, we will have two COM ports active - one for Application Core logs and one for Network Core logs.

@kapi-no done

Reply:
Thank you, please also update ``nrf5340_cpunet_reset.c `` file. Pin assignments still use the old UART configuration since they are not derived from DTS. Please see the linked commit in this discussion if you need further reference.

Reply:
Yes, I understand. Sure, I will have to change this too.

Reply:
Done now @kapi-no thanks for the input.

Reply:
Happy to help :) This PR looks good to me now, I have no further comments